plugin/stripcomments/checks.go

```diff
@@ -0,0 +1,50 @@
+package stripcomments
+
+import (
+	"slices"
+
+	"github.com/safedep/code/core"
+	sitter "github.com/smacker/go-tree-sitter"
+)
+
+type isCommentNodeCheck func(node *sitter.Node) bool
+
+// Verify contract
+var _ isCommentNodeCheck = isJavascriptCommentNode
+
+func isJavascriptCommentNode(node *sitter.Node) bool {
+	return node.Type() == "comment"
+}
+
+// Verify contract
+var _ isCommentNodeCheck = isPythonCommentNode
```

Comment:
Keep contract verification assertions together. Help improve readability. Its like `checks.go` starting with a list of all language specific check functions adhering to a specific contract

Reply:
ack, updated